README.md

```diff
@@ -1,90 +1,68 @@
 Zydis Pascal Bindings
 =====================
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![Gitter](https://badges.gitter.im/zyantific/zyan-disassembler-engine.svg)](https://gitter.im/zyantific/zydis?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge) [![Discord](https://img.shields.io/discord/390136917779415060.svg)](https://discordapp.com/channels/390136917779415060/390138781313007626) 
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
 Pascal language bindings for the [Zydis library](https://github.com/zyantific/zydis), a fast and lightweight x86/x86-64 disassembler.
 
-## Readme
-The Zydis pascal bindings support both dynamic and static linkage. The required binaries for Windows can be acquired in the [Releases](https://github.com/zyantific/zydis-pascal/releases) section.
+## TODO
+This repo still in development & Supports only FPC for now
```

Comment:
I'm happy to provide the Delphi projects. We should consider going with `*.dpr`, `*.dfm`, .etc as AFAIK they can be opened with FPC/Lazarus as well but the other way is not possible.

Reply:
I do have Delphi, but I'll do most of the work on FPC first and then test it with Delphi.
I'll change the `.lpr` to `.dpr`